openstack/baremetal/v1/nodes/requests.go

```diff
@@ -9,37 +11,38 @@ import (
 // List request.
 type ListOptsBuilder interface {
 	ToNodeListQuery() (string, error)
+	ToNodeListDetailQuery() (string, error)
 }
 
 // Provision state reports the current provision state of the node, these are only used in filtering
 type ProvisionState string
 
 const (
 	Enroll       ProvisionState = "enroll"
-	Verifying    ProvisionState = "verifying"
-	Manageable   ProvisionState = "manageable"
-	Available    ProvisionState = "available"
-	Active       ProvisionState = "active"
-	DeployWait   ProvisionState = "wait call-back"
-	Deploying    ProvisionState = "deploying"
-	DeployFail   ProvisionState = "deploy failed"
-	DeployDone   ProvisionState = "deploy complete"
-	Deleting     ProvisionState = "deleting"
-	Deleted      ProvisionState = "deleted"
-	Cleaning     ProvisionState = "cleaning"
-	CleanWait    ProvisionState = "clean wait"
-	CleanFail    ProvisionState = "clean failed"
-	Error        ProvisionState = "error"
-	Rebuild      ProvisionState = "rebuild"
-	Inpsecting   ProvisionState = "inspecting"
-	InspectFail  ProvisionState = "inspect failed"
-	InspectWait  ProvisionState = "inspect wait"
-	Adopting     ProvisionState = "adopting"
-	AdoptFail    ProvisionState = "adopt failed"
-	Rescue       ProvisionState = "rescue"
-	RescueFail   ProvisionState = "rescue failed"
-	Rescuing     ProvisionState = "rescuing"
-	UnrescueFail ProvisionState = "unrescue failed"
+	Verifying                   = "verifying"
```

Comment:
I'm not sure I really get why this was changed in this commit..

Reply:
Unrelated style change, this is all going to get squashed anyway...